### PR TITLE
Support enterprise version and show current branch only

### PR DESCRIPTION
**Pull request by doyou89**
Add additional features
[x] Support CircleCI enterprise: add 'URL' configuration (#17)
[x] Show current branch only: filtering builds by the current branch if the project based on git (#13)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
   "name": "circleci-status",
   "displayName": "CircleCI Status",
   "description": "CircleCI Status is a VSCode extension to check CircleCI build status.",
-  "version": "1.0.5",
+  "version": "1.1.0",
   "publisher": "kuromoka",
   "license": "https://github.com/kuromoka/circleci-status/blob/master/LICENSE",
-  "homepage": "https://github.com/kuromoka/circleci-status/blob/master/README.md",
+  "homepage": "https://github.com/doyou89/circleci-status/blob/master/README.md",
```
Comment (kuromoka) [inline package.json:8]: please leave it as it is.
https://github.com/kuromoka/circleci-status/blob/master/README.md

**Comment by kuromoka** [comment]
@doyou89 thanks for contributing!

File: package.json (hunk 1 above)
Comment (doyou89) [reply package.json]: back to the original homepage in 752b721 commit.

**Comment by doyou89** [comment]
When this can be merged? Are there any more things to do?

**Comment by doyou89** [comment]
merge please

**Pull request merged**